### PR TITLE
Feat: Toolkits

### PR DESCRIPTION
**Pull request by levanion**

### CODE REVIEW

File: apps/ui/src/services/config/useUpdateConfigService.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export const useUpdateConfigService = () => {
           is_required,
           datasource_id: datasource_id,
           agent_id: null,
-          tool_id: null,
+          toolkit_id: tool_id,
```
Comment (Chkhikvadze) [inline apps/ui/src/services/config/useUpdateConfigService.ts:22]: is i right here to set tool_id to toolkit_id

Comment (Chkhikvadze) [inline apps/ui/src/services/config/useUpdateConfigService.ts:22]: is it right to set tool_id to toolkit_id

Comment (Chkhikvadze) [reply apps/ui/src/services/config/useUpdateConfigService.ts:22]: @okradze @levanion

**Review by Chkhikvadze — APPROVED** [review]

File: apps/ui/src/services/tool/useToolsService.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { useQuery } from '@apollo/client'
+import toolsGql from '../../gql/ai/tool/tools.gql'
```
Comment (okradze) [inline apps/ui/src/services/tool/useToolsService.ts:2]: toolkits.gql

and this file should be toolkit service

File: apps/ui/src/pages/Toolkit/constants.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
+export const toolLogos = [
```
Comment (okradze) [inline apps/ui/src/pages/Toolkit/constants.ts:1]: use uppercase for constants

**Pull request merged**